### PR TITLE
Fixed the jar name for the aspectjweaver

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

The jar names in `libs` folder were renamed to include the artifact group id (for better identification). 

When we start the zookeeper process, we need to pass the right jar for AspectjWeaver in order to enabled detailed ZK stats.

### CODE REVIEW

File: bin/pulsar [hunk 1]
```diff
@@ -204,15 +204,15 @@ elif [ $COMMAND == "zookeeper" ]; then
     PULSAR_LOG_FILE=${PULSAR_LOG_FILE:-"zookeeper.log"}
 
     # Add instrumentation
-    WEAVER_JAR=`find $PULSAR_HOME/lib -name 'aspectjweaver-*.jar'`
+    WEAVER_JAR=`find $PULSAR_HOME/lib -name 'org.aspectj-aspectjweaver-*.jar'`
```
Comment (rdhabalia) [inline bin/pulsar]: a small suggestion: can't we load it dynamically in `ZooKeeperStarter` similar way we are doing at [PulsarBrokerStarter](https://github.com/apache/incubator-pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/PulsarBrokerStarter.java#L60)

Comment (merlimat) [reply bin/pulsar]: Good point. That's much easier.

**Comment by rdhabalia** [comment]
retest this please

**Comment by rdhabalia** [comment]
retest this please

**Pull request merged**